models/marts/wash/wash_marts_test.yml

```diff
@@ -1,66 +0,0 @@
-version: 2
-
-models:
-  - name: wash_facilities
-    description: "Select all columns from facilities table as base data for wash facilities analysis"
-    columns:
```

Comment:
Same question here, why are we removing these tests ?

Reply:
We restructured models, so they're irrelevant now